src/components/standalone/dns_dhcp/DynamicLeases.vue

```diff
@@ -69,6 +69,13 @@ const filteredItems = computed<DynamicLease[]>(() => {
   return result
 })
 
+function formatTimestamp(ts: string): string {
+  const num = Number(ts)
+  if (!num) return ''
```

Comment:
I'll 100% miss this if not using an editor :sweat_smile: 
```suggestion
  if (!num) {
    return ''
  }
```